src/main/features/core/applicationMenu.js

```diff
@@ -125,7 +125,10 @@ const template = [
       },
       {
         label: 'Close',
-        accelerator: 'CmdOrCtrl+W',
+        accelerator: (() => {
```

Comment:
Ah sorry, I thought you meant that as a general guideline and noticed you ad already done something similar, so tried to keep it consistent. I'll do another one real quick.